src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metrics_grid_section.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { MetricsGrid } from './metrics_grid';
+import { MetricField } from '../types';
+
+export const MetricsGridSection: React.FC<{
```

Comment:
This looks like a totally redundant component, but the idea is to include the other section content like the header, metrics count, etc. ( possibly the pagination )  and the hardcoded values will be moved to the props once we have the APIs available. 
If we see that this component is not extendable in this form, it could be removed/refactored